### PR TITLE
feat(ci): use kaniko + harbor

### PR DESCRIPTION
**Pull request by revolunet**
same as https://github.com/SocialGouv/code-du-travail-numerique/pull/3461

NB : both projects publish images into the same `HARBOR_PROJECT` (cdtn)

So we need to be careful with the images names (`cdtn-admin-hasura` vs `hasura`)

### CODE REVIEW

File: targets/frontend/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,6 @@ RUN  npx @socialgouv/yarn-workspace-focus-install \
   --cwd targets/frontend \
   --production \
   -- \
-  --cache-folder /dev/shm/yarn \
   --frozen-lockfile \
   --prefer-offline
```
Comment (douglasduteil) [inline targets/frontend/Dockerfile:21]: We might want to manually erase the cache then ` && yarn cache clean `

**Review by douglasduteil — APPROVED** [review]
LGTM

File: targets/frontend/Dockerfile (hunk 1 above)
Comment (lionelB) [reply targets/frontend/Dockerfile:21]: il faut rajouter ce commit avant de merger ?

**Review by m-maillot — APPROVED** [review]

File: targets/frontend/Dockerfile (hunk 1 above)
Comment (revolunet) [reply targets/frontend/Dockerfile:21]: Ca a l'air ajouté

**Pull request merged**